### PR TITLE
Creates a New Workflow for Adding Wiki Labels  

### PR DESCRIPTION
**Pull request by Burzah**
## What Does This PR Do
This creates a new workflow that allows people designated to label a PR with `Requires Wiki Update` without needing to be a member of the repository. This can be done by an authorized user by using `!wiki_label`.

~~Additionally this PR adds @SynthTwo to the list of authorized users for this command.~~

This now uses a repository variable and will need to be setup accordingly.

## Why It's Good For The Game
What is good for the code base is good for the game.

## Images of changes
librewolf_FyKE82QVds
librewolf_9SSv4nbSAQ

## Testing
Created a separate fork of our repository.
Created a dummy PR to test the function of the workflow.
Tested new changes made during the course of this PR.

Additional thanks to @SynthTwo for helping to test this.

## Changelog
NPFC

### CODE REVIEW

File: .github/workflows/wiki_label.yml
Comment (farie82) [inline .github/workflows/wiki_label.yml:51]: I think this can be removed and only post a comment if the user is authorized.

Comment (Burzah) [reply .github/workflows/wiki_label.yml:51]: Would you be amenable to removing the comment posting, but leaving the thumbs down reaction if rejected?

File: .github/workflows/wiki_label.yml
Comment (farie82) [inline .github/workflows/wiki_label.yml:21]: Could you make this into a list? A list of one for now, but makes it easier to expand later.

File: .github/workflows/wiki_label.yml
Comment (Contrabang) [reply .github/workflows/wiki_label.yml:51]: IMO, less comment bloat the better, I think reactions are good enough for both the authorized and unauthorized cases

Comment (Burzah) [reply .github/workflows/wiki_label.yml:51]: I can agree with that. I wasn't really sure about adding the additional messages to begin with. Considering as soon as the bot does accept a valid command, it will just apply the label and thumbs up the comment. Plus, I don't really see the need now for rejection message either, since the authorized user pool will be small regardless.

Comment (farie82) [reply .github/workflows/wiki_label.yml:51]: A thumbsup on the original message is better. Or just leave out the comments altogether since indeed they will apply the change requested as a response

File: .github/workflows/wiki_label.yml
Comment (farie82) [inline .github/workflows/wiki_label.yml:21]: We can make this into a repo variable instead. So HCs can tweak it easier without having to do a code change. I've made one with the name `WIKI_MANAGERS`
https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository
Then we can use the following to get the list from the string value:
https://stackoverflow.com/questions/75391000/define-list-of-elements-in-a-github-action-variable
And check if the username is in the list of allowed usernames
https://stackoverflow.com/questions/65653291/conditional-to-check-if-a-string-is-included-in-a-list

File: .github/workflows/wiki_label.yml
Comment (Burzah) [reply .github/workflows/wiki_label.yml:51]: Yup! I am just going to remove the comments as suggested, and leave the thumbs up and down. I think that is best.

File: .github/workflows/wiki_label.yml
Comment (Burzah) [reply .github/workflows/wiki_label.yml]: I have made the changes, but I will have to test this again locally via my fork to make sure it works properly.

File: .github/workflows/wiki_label.yml
Comment (AffectedArc07) [inline .github/workflows/wiki_label.yml:15]: Do we need to checkout the repo for a comment action?

Comment (Burzah) [reply .github/workflows/wiki_label.yml:15]: Not anymore.
I will take it out on the next commit.

Comment (Burzah) [reply .github/workflows/wiki_label.yml]: New changes pushed. I will test locally again to make sure it still functions properly.

File: .github/workflows/wiki_label.yml
Comment (Burzah) [reply .github/workflows/wiki_label.yml]: We are now using the WIKI_MANAGERS repository variable for this. Testing to be done locally.

File: .github/workflows/wiki_label.yml
Comment (S34NW) [inline .github/workflows/wiki_label.yml:29]: Do we need this line for when it is added, i think it might be clear enough when the label is just added?

Comment (Burzah) [reply .github/workflows/wiki_label.yml:29]: This is for the workflow itself, I removed the comments. `echo` here  is what will show up when you looking at the workflow actions themselves. But yeah, the consensus was that the thumbs up and down and the label being added would just be enough.

**Review by S34NW — APPROVED** [review]

File: .github/workflows/wiki_label.yml
Comment (Burzah) [reply .github/workflows/wiki_label.yml]: This has now been tested and updated. 
Repository variable instructions and updating to be handled internally.

**Review by farie82 — APPROVED** [review]
Waiting for the policy change after that this should be good to go!
EDIT:
Policy change got merged, this can be merged once it is approved by @AffectedArc07

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**
